projects/graph/graph.py

```diff
@@ -3,46 +3,58 @@
 """
 from util import Stack, Queue  # These may come in handy
 
+
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
+
     def __init__(self):
         self.vertices = {}
+
     def add_vertex(self, vertex):
-        """
-        Add a vertex to the graph.
-        """
-        pass  # TODO
+        self.vertices[vertex] = set()
```

Comment:
Good use of a comment here to explain the _what_ of a `set` Michael 👍 . Depending on the implementation details of your graph you could use a more descriptive name for your `vertex` param. It could be an `id`, `name`, `label` etc..etc...